### PR TITLE
Update 0001-codable-routing.md

### PR DESCRIPTION
**Pull request by tunniclm**
* Minor formatting/indent changes
* Removed extraneous `init` in struct (you get one automatically if you have no other `init` defined)
* Corrected JSON return value
* Alternative approaches to completion callback

### CODE REVIEW

File: Kitura/Proposals/0001-codable-routing.md [hunk 1]
```diff
@@ -76,10 +76,6 @@ Utilizing the new API layer, the above code becomes:
 ```swift
 struct Name: Codable {
    let name: String
-   
-   init(name: String) {
```
Comment (seabaylea) [inline Kitura/Proposals/0001-codable-routing.md:80]: This is only superfluous if you're only going to allow instances to be created using `JSONDecoder().decode()` - which I guess is true in our example ;-)

Comment (tunniclm) [reply Kitura/Proposals/0001-codable-routing.md:80]: You get a default contructor for structs that matches the stored properties. Check it out: http://swift.sandbox.bluemix.net/#/repl/59e5f9921b6536583365e490

Comment (seabaylea) [reply Kitura/Proposals/0001-codable-routing.md:80]: TIL ;-)

**Comment by seabaylea** [comment]
LGTM. Merging.

**Pull request merged**